growthstreet_zoo/zoo.py

```diff
@@ -146,15 +103,23 @@ class BaseAnimal(object):
     QUALITY_TOUGH = 'TO'
     QUALITY_TENDER = 'TE'
 
-    def __init__(self, name, cage=None):
+    COMPETITION_MAP = {'Lion': 1,
```

Comment:
This is an interesting way to model the tastes of different animals, but it's not really following the original scope.  Can you think of a simpler interpretation?  Let's also discuss how the processes we might follow if we feel a more complex model is needed, or if the scope isn't clear (which in real life is often!).